src/Map/src/AssetMapper/ImportMap/Compiler/LeafletReplaceImagesAssetCompiler.php

```diff
@@ -0,0 +1,66 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\UX\Map\AssetMapper\ImportMap\Compiler;
+
+use Psr\Log\LoggerInterface;
+use Symfony\Component\AssetMapper\AssetMapperInterface;
+use Symfony\Component\AssetMapper\Compiler\AssetCompilerInterface;
+use Symfony\Component\AssetMapper\Exception\RuntimeException;
+use Symfony\Component\AssetMapper\MappedAsset;
+use Symfony\Component\Filesystem\Path;
+
+/**
+ * Replaces the image paths in the Leaflet library JavaScript code with their public path.
+ *
```

Comment:
Add "@internal" :)

Reply:
Also added in `LeafletPackageResolver.php`